src/renderer/components/watch-video-comments/watch-video-comments.js

```diff
@@ -120,6 +121,25 @@ export default defineComponent({
 
     subscriptions: function() {
       return this.$store.getters.getActiveProfile.subscriptions
+    },
+
+    highlightedComments: function () {
+      return this.$store.getters.getHighlightedComments(this.id)
+    },
+    highlightedReplyComments: function () {
+      return this.$store.getters.getHighlightedReplyComments(this.id)
+    },
+    isCommentHighlighted: function () {
+      return (comment) => !!this.$store.getters.getHighlightedComments(this.id).find((entry) => entry.commentId === comment.commentId)
+    },
+    isReplyHighlighted: function () {
+      return (comment, reply) => !!this.$store.getters.getHighlightedReplies(this.id, comment.commentId).find(entry => entry.commentId === reply.commentId)
+    },
+    getHighlightedReplies: function () {
+      return (commentId) => this.$store.getters.getHighlightedReplies(this.id, commentId)
+    },
+    isCommentOrReplyHighlighted: function () {
+      return (comment) => this.isCommentHighlighted(comment) || this.getHighlightedReplies(comment.commentId).length > 0
```

Comment:
Methods should be added to the `methods` section, not returned from computed properties. I'm guessing this is your first time coding with Vue, if yes you may want to read up on the options API, the official Vue 2 docs are a good place to start.